Makefile

```diff
@@ -0,0 +1,54 @@
+KERNEL_OUTPUT=$(CURDIR)/output/vmlinux.container
+IMAGE_OUTPUT=$(CURDIR)/output/container.img
+
+ifdef http_proxy
+PROXY=--build-arg http_proxy=$(http_proxy)
+RUN_PROXY=--env http_proxy=$(http_proxy)
+endif
+
+ifdef https_proxy
+PROXY+=--build-arg https_proxy=$(https_proxy)
+RUN_PROXY+=--env https_proxy=$(https_proxy)
+endif
+
+IMG_SIZE=128M
+
+image: docker_build output
+	docker run \
+		--runtime runc \
+		--privileged \
+		-v /dev:/dev \
+		$(RUN_PROXY) \
+		-ti \
+		-v $(CURDIR)/image_creator.sh:/image_creator.sh \
+		-v $(CURDIR)/output:/output \
+		-v $(CURDIR)/rootfs:/content \
+		cc-image-builder \
+		bash -c "/image_creator.sh $(IMG_SIZE); cp container.img /output"
+		@echo New image created: $(IMAGE_OUTPUT)
+
+docker_build: 
+	docker build $(PROXY) -t cc-image-builder . 
+
+build_rootfs:
+	docker run -ti  \
+		-v $(CURDIR)/rootfs:/rootfs \
+		-v $(CURDIR)/yum.conf:/yum.conf \
+		-v $(CURDIR)/build_cc_rootfs.sh:/build_cc_rootfs.sh \
+		$(RUN_PROXY) \
+		fedora:25 \
+		bash /build_cc_rootfs.sh
```

Comment:
I would like this to be a little simpler:

* Have one single script that could either build a rootfs, a kernel, or a complete image depending on an option you pass it.
* Have one Dockerfile that uses that script as its ENTRYPOINT
* Call `docker run -d <rootfs|kernel|image>` on the built container

Basically, it would be better for the logic to be moved out of the Makefile, into the Dockerfile. Make the Makefile a `docker <build|run>` wrapper.